semantic/tools/semantic2vef.cpp

```diff
@@ -235,6 +235,11 @@ int Semantic2Vef::run()
         geometry::Mesh mesh = semantic::mesh(world, meshConfig_);
         addTextureCoords(mesh, txtColorWidth_);
 
+        auto ext3 { math::transform(
+            worldTf,
+            math::computeExtents(mesh.vertices.begin(), mesh.vertices.end())) };
```

Comment:
you can just use `math::extents(mesh.vertices)` instead of iterator-based function

Reply:
True, I didn't notice. Fixed